setup.py

```diff
@@ -35,7 +35,7 @@
         [console_scripts]
         linchpin=linchpin.shell:runcli
     ''',
-    tests_require=["pytest", "nose", "mock", "coverage", "flake8"],
+    tests_require=["pytest<=4.4.0", "nose", "mock", "coverage", "flake8"],
```

Comment:
What changed in newer versions of pytest?

Reply:
Didn't actually check on that. But its currently failing on on latest version.